### DISCUSSION TITLE
Debian repos

### DISCUSSION THREAD

**Discussion by GregJohnStewart** [kasmtech/KasmVNC#230] [Ideas]
What's the consensus on setting up repos for Kasm? Just the installers are nice, but it's actually pretty easy to setup a debian repo. Can help if needed.

**Comment by mmcclaskey** [comment]
I've considered it, just don't have the time. We would definitely take your assistance. I did a quick read and it looks like it will just need a static web server. So what would likely be helpful is a bash script that creates the directory structure, downloads the deb releases for all the operating systems we support that can use an apt repo for both arm and amd64, put them in the appropriate directory, do the pgp signing stuff (self signed for testing), and create the required Release file. 

So we should just be able to run the script you create and then point NGINX at the target directory. After testing, we would likely take the script and put it into a pipeline and have the output written to S3 and use CloudFront to host behind our domain.

**Reply by GregJohnStewart** [reply]
Sure thing, just let me know where you need me. Once you get the initial debian repo setup it's fairly trivial to update with a script as you say.

I have a project I run a script that pulls the installers from the gitbub releases, then publishes them to pages. It seems to work pretty well, and you don't need to mess with passing it back to your other stack(s). Just an idea.
